senaite-openmrs/src/test/resources/application-test.properties

```
@@ -16,106 +21,126 @@ db-event.retry.initial.delay=1000
 # connector property named database.password
+debezium.db.password=${MYSQL_ADMIN_USER_PASSWORD}
 
-##debezium.db.password=test
+# The interval in milliseconds before the first poll of the debezium event queue by the debezium reader route when the
+# application starts.
+debezium-event-reader.initial.delay=${DEBEZIUM_EVENT_READER_INITIAL_DELAY:1000}
 
-# Specifies the criteria for running a snapshot when the connector starts, DO NOT change after setting it, maps to the
-# debezium MySQL connector property named snapshot.mode
-
-##debezium.snapshotMode=schema_only
-
-# The interval in milliseconds before the first poll for database events
-debezium-event-reader.initial.delay=1000
-
-# The interval in milliseconds between polls of reading database events
-debezium-event-reader.repeat.interval=100
+# The interval in milliseconds between polls of the debezium event queue by the debezium reader route
+debezium-event-reader.repeat.interval=${DEBEZIUM_EVENT_READER_REPEAT_INTERVAL:1000}
 
 # The 2 properties below are debezium engine specific and are documented at
 # https://debezium.io/documentation/reference/development/engine.html#engine-properties
 #
 # Path to file where offsets are to be stored, maps to property named offset.storage.file.filename, DO NOT change after
 # setting it
-debezium.offsetFilename=offsets-test.txt
+debezium.offsetFilename=${DEBEZIUM_OFFSET_FILENAME:.offsets.txt}
 
 # Path to file where offsets are to be stored, maps to property named database.history.file.filename, DO NOT change
 # after setting it
-debezium.historyFilename=dbhistory-test.txt
+debezium.historyFilename=${DEBEZIUM_HISTORY_FILENAME:.history.txt}
+
+# Maximum size of each batch of source records to be read by debezium from the binlog files, maps to the
+## debezium MySQL connector property named max.batch.size
+#https://debezium.io/documentation/reference/1.6/connectors/mysql.html#mysql-property-max-batch-size
+debezium.reader.maxBatchSize=${DEBEZIUM_READER_MAX_BATCH_SIZE:2048}
+
+# Specifies the criteria for running a snapshot when the connector starts, DO NOT change after setting it, maps to the
+# debezium MySQL connector property named snapshot.mode
+debezium.snapshotMode=schema_only
+
+debezium.snapshotLockingMode=extended
+# ----------------------------------------------------------------------------------------------------------------------
+
+# *********************** OpenMRS FHIR EIP Configuration ***************************************************************
+#
+# A comma separated list of FHIR resources, e.g., Patient, Encounter, Observation
+eip.fhir.resources=${EIP_FHIR_RESOURCES:Patient,ServiceRequest}
+# The URL of the FHIR server
+eip.fhir.serverUrl=${EIP_FHIR_SERVER_URL:http://localhost:8080/openmrs/ws/fhir2/R4}
+
+# FHIR server credentials
+eip.fhir.username=${EIP_FHIR_USERNAME:admin}
+eip.fhir.password=${EIP_FHIR_PASSWORD:Admin123}
```

```suggestion
eip.fhir.username=admin
eip.fhir.password=Admin123
```
Are you intending to use environment variables in your tests? If not, they might be redundant since the default values will always be used. 

For example, in your configuration:

```properties
eip.fhir.username=${EIP_FHIR_USERNAME:admin}
eip.fhir.password=${EIP_FHIR_PASSWORD:Admin123}
```

If `EIP_FHIR_USERNAME` and `EIP_FHIR_PASSWORD` are not set in the test environment, the application will default to using "admin" and "Admin123" respectively. If the intention is to always use these default values, it's simpler and clearer to write them directly:

```properties
eip.fhir.username=admin
eip.fhir.password=Admin123
```

However, if you plan to override these values in the test environment to ensure that `EIP_FHIR_USERNAME` and `EIP_FHIR_PASSWORD` are set appropriately, then including the env vars in `application-test.properties` is fine otherwise we should do away with it. This applies to other environment variables.

---

Will create another PR to clean this up